REFERENCE.md

```diff
@@ -30,6 +30,9 @@ Table of contents
   + [Mapping bindings](#mapping-bindings)
     - [Example use of `mapModel` and `mapMsg`](#example-use-of-mapModel-and-mapMsg)
     - [Theory behind `mapModel` and `mapMsg`](#theory-behind-mapModel-and-mapMsg)
+* [Statically-typed view models](#statically-typed-view-models)
+  + [Inherit from `ViewModelBase<'model, 'msg>`]()
+  + [Typed Bindings]()
```

Comment:
Shouldn't we add links to the sections?

Reply:
Yes I published this too soon. Can you push a commit with those changes? Or do you not have access to my branch :\

Reply:
Nevermind I got it